### PR TITLE
[INLONG-8462][Manager] Fix attribute was overwritten by the default value error when modifying the group

### LINKED ISSUE #8462: [Bug][Manager] Attribute was overwritten by the default value error when modifying the group
### What happened

 Attribute was overwritten by the default value error when modifying the group.

### What you expected to happen

 Attribute was overwritten by the default value error when modifying the group.

### How to reproduce

 Attribute was overwritten by the default value error when modifying the group.

### Environment

_No response_

### InLong version

master

### InLong Component

InLong Manager

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fuweng11**
### Prepare a Pull Request

- Fixes #8462 

### Motivation

Fix attribute was overwritten by the default value error when modifying the group.

### Modifications

Fix attribute was overwritten by the default value error when modifying the group.

### CODE REVIEW

**Review by gosonzhang — APPROVED** [review]

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml [hunk 1]
```diff
@@ -65,28 +65,149 @@
...
+                #{maxLength,jdbcType=INTEGER},
+            </if>
+            <if test="enableZookeeper != null">
+                #{enableZookeeper,jdbcType=INTEGER},
+            </if>
+            <if test="enableCreateResource != null">
+                #{enableCreateResource,jdbcType=INTEGER},
+            </if>
+            <if test="lightweight != null">
+                #{lightweight,jdbcType=INTEGER},
```
Comment (gong) [inline inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml:181]: field is inconsitent

**Review by dockerzhang — APPROVED** [review]

**Comment by gong** [comment]
@fuweng11 I don't understand why is overwritten. Beause you just modify insert.

File: inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml (hunk 1 above)
Comment (fuweng11) [reply inlong-manager/manager-dao/src/main/resources/mappers/InlongGroupEntityMapper.xml]: Fix.

**Comment by fuweng11** [comment]
@gong Because the default value was originally set in the request, when the user did not set the value, the default value will overwrite the correct value in the database when the modification operation is performed.

**Review by gong — APPROVED** [review]

**Pull request merged**